### ISSUE TITLE
Jump to PDF functionality

### ISSUE THREAD

**Issue by LaurenzV** [typst/typst#2150]
### Description

There already is a feature that allows you to click somewhere in the preview and it leads you to where it is in the source code, which is very useful! However, having the opposite would be great as well, e.g. given that you are somewhere in the code, it would be great if there were some functionality that briefly highlights the position in the preview panel.

### Use Case

It makes it easier to find where certain parts of the code are located in the actual output.

**Comment by laurmaedje** [comment]
This does exist in some form. When you type, it scrolls the relevant part of the PDF into view. Having an explicit highlight on top could be nice though.

**Comment by LaurenzV** [comment]
Yeah, having it jump into view already is pretty nice, but I'm currently rewriting my bachelor thesis in Typst and having a way of finding _where exactly_ in the Preview the text is is something I missed.
